### PR TITLE
CVSB-3022

### PR DESCRIPTION
**Pull request by mgherceanu**

### CODE REVIEW

File: src/pages/visit/visit-timeline/visit-timeline.ts [hunk 1]
```diff
@@ -21,6 +21,7 @@ export class VisitTimelinePage implements OnInit {
   loading = this.loadingCtrl.create({
     content: APP_STRINGS.END_VISIT_LOADING
   });
+  count: number = 0;
```
Comment (danieliordache) [inline src/pages/visit/visit-timeline/visit-timeline.ts]: remove this line

File: src/pages/testing/test-creation/test-review/test-review.ts [hunk 2]
```diff
@@ -24,6 +24,7 @@ import { TestService } from "../../../../providers/test/test.service";
 import { Observable } from "rxjs";
 import { OpenNativeSettings } from "@ionic-native/open-native-settings";
 import { VisitService } from "../../../../providers/visit/visit.service";
+import { take, tap } from "rxjs/operators";
```
Comment (danieliordache) [inline src/pages/testing/test-creation/test-review/test-review.ts]: remove "take" import

File: src/pages/visit/visit-timeline/visit-timeline.ts (hunk 1 above)
Comment (mgherceanu) [reply src/pages/visit/visit-timeline/visit-timeline.ts]: done

File: src/pages/testing/test-creation/test-review/test-review.ts (hunk 2 above)
Comment (mgherceanu) [reply src/pages/testing/test-creation/test-review/test-review.ts]: done

**Pull request merged**